docs/resources/drs_job.md

```diff
@@ -177,7 +177,9 @@ The `db_info` block supports:
 * `region` - (Optional, String, ForceNew) Specifies the region which the database belongs when it is a RDS database.
  Changing this parameter will create a new resource.
 
-* `ssl_enabled` - (Optional, String, ForceNew) Specifies whether to enable SSL connection.
+* `name` - (Required, String, ForceNew) Specifies the name of database. Changing this parameter will create a new resource.
```

Comment:
it seems that this parameter is optional in schema, please check it.

Reply:
yes, it should be `Optional`.